src/models/MyAccount.ts

```diff
@@ -114,18 +131,25 @@ export default class MyAccount {
   buildHolds(holds: SierraHold[], bibData: SierraBibEntry[]): Hold[] {
     const bibDataMap = MyAccount.buildBibData(bibData)
     return holds.map((hold: SierraHold) => {
+      // Hold without bibIds is a bib level id.
+      const bibId = hold.record.bibIds ? hold.record.bibIds[0] : hold.record.id
       return {
         patron: MyAccount.getRecordId(hold.patron),
         id: MyAccount.getRecordId(hold.id),
         pickupByDate: MyAccount.formatDate(hold.pickupByDate) || null,
         canFreeze: hold.canFreeze,
         frozen: hold.frozen,
         status: MyAccount.getHoldStatus(hold.status),
-        pickupLocation: hold.pickupLocation.name,
-        title: bibDataMap[hold.record.bibIds[0]].title,
-        isResearch: bibDataMap[hold.record.bibIds[0]].isResearch,
-        bibId: hold.record.bibIds[0],
-        isNyplOwned: bibDataMap[hold.record.bibIds[0]].isResearch,
+        pickupLocation: hold.pickupLocation,
+        title: bibDataMap[bibId].title,
```

Comment:
Assigning `bibDataMap[bibId]` to a variable here (ie `bibForHold`) could make it clearer what object we're looking at